proposed/http-message.md

```diff
@@ -40,9 +40,9 @@ This is the response body
 ```
 
 The first line is the "status line", and contains, in order, the HTTP protocol
-version, the HTTP status code, and a "reason phrase," a human-readable
-description of the status code. Just like the request message, this is then
-followed by one or more HTTP headers, and empty line, and the message body.
+version, the HTTP status code, and a "reason phrase", a human-readable
```

Comment:
Original was correct; punctuation should be inside quotes. (See Strunk & White or the Chicago Manual of Style.)